### DISCUSSION TITLE
V 1.4.0

### DISCUSSION THREAD

**Discussion by gickowtf** [gickowtf/pixoo-homeassistant#5] [Announcements]
Hey guys, I'm working on the integration again as time permits. 
I have planned the following for the next update:

- [X] Brightness for the LightSensor
- [ ] scrolling text for texts page
- [ ]  possibly a gaming scoreboard
- [x]  possibly new font

If you have further wishes, I will possibly respond to them depending on the effort.
The next update will come at the earliest when this repo reaches 16 stars. <- personal milestone ;)

Well then, best regards gickowtf

**Comment by wacholderpolka** [comment]
This is a really amazing project!
Many thanks for your work.I just got my photovoltaic running, homeassistant is also setup and some data from the energy system integrated.
Then I found your project and now I am about to buy the Pixoo for just this integration:-)
I tried to find the repo in HACS but it is not found, do I miss something (searched for "Pixoo", "Divoom")?

**Reply by gickowtf** [reply]
Installation

Install this integration with HACS (adding repository required), or copy the contents of this repository into the custom_components/pixoo-homeassistant directory.
Restart Home Assistant.

**Reply by wacholderpolka** [reply]
I got it working. I did not realise, that the integration search was not the right way, so I found the (not so obvious) way to add the github path to the integration list :-)
Many thanks again, so nice to play around with it.

**Comment by dadiggi** [comment]
Hi,

   great work here! Same as previous comment, I was also in the state of weighing whether Pixoo is worth buying or not, especially in HA environment.
It's a cool gadget, but on the other hand it is not cheap so I'd expect to get some more from it.
And I bumped to your add-on and  decided to go for it! ;)
Anyway, looking forward for the new version!

**Comment by brachycera** [comment]
Hello,
>     * [ ]  scrolling text for texts page
I would love to see this feature in the coming releases. This would be super helpful to show long text strings, especially when displaying a Song Title and Artists.
Thanks for the nice integration 🥳
